[2022-05-15 21:56:45,563] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.fct_customer_orders scheduled__2022-05-14T16:00:47.250021+00:00 [queued]>
[2022-05-15 21:56:45,575] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.fct_customer_orders scheduled__2022-05-14T16:00:47.250021+00:00 [queued]>
[2022-05-15 21:56:45,576] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-15 21:56:45,576] {taskinstance.py:1242} INFO - Starting attempt 3 of 3
[2022-05-15 21:56:45,577] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-15 21:56:45,590] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_run.model.jaffle_shop.fct_customer_orders> on 2022-05-14 16:00:47.250021+00:00
[2022-05-15 21:56:45,594] {standard_task_runner.py:52} INFO - Started process 293 to run task
[2022-05-15 21:56:45,599] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'model_level_dbt_dag', 'dbt_run.model.jaffle_shop.fct_customer_orders', 'scheduled__2022-05-14T16:00:47.250021+00:00', '--job-id', '265', '--raw', '--subdir', 'DAGS_FOLDER/model_level_dbt_dag.py', '--cfg-path', '/tmp/tmps6xj3vvv', '--error-file', '/tmp/tmpjz7zjvnq']
[2022-05-15 21:56:45,600] {standard_task_runner.py:77} INFO - Job 265: Subtask dbt_run.model.jaffle_shop.fct_customer_orders
[2022-05-15 21:56:45,649] {logging_mixin.py:109} INFO - Running <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.fct_customer_orders scheduled__2022-05-14T16:00:47.250021+00:00 [running]> on host 9922a9d65d01
[2022-05-15 21:56:45,707] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=model_level_dbt_dag
AIRFLOW_CTX_TASK_ID=dbt_run.model.jaffle_shop.fct_customer_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-05-14T16:00:47.250021+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-14T16:00:47.250021+00:00
[2022-05-15 21:56:45,709] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-15 21:56:45,710] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt --no-write-json run --target dev --models fct_customer_orders --profiles-dir /opt/***/dags/jaffle_shop --project-dir /opt/***/dags/jaffle_shop']
[2022-05-15 21:56:45,724] {subprocess.py:85} INFO - Output:
[2022-05-15 21:56:49,210] {subprocess.py:89} INFO - 21:56:49  Running with dbt=1.1.0
[2022-05-15 21:56:49,424] {subprocess.py:89} INFO - 21:56:49  Found 5 models, 13 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-15 21:56:49,435] {subprocess.py:89} INFO - 21:56:49
[2022-05-15 21:56:49,604] {subprocess.py:89} INFO - 21:56:49  Concurrency: 1 threads (target='dev')
[2022-05-15 21:56:49,607] {subprocess.py:89} INFO - 21:56:49
[2022-05-15 21:56:49,621] {subprocess.py:89} INFO - 21:56:49  1 of 1 START table model dev.fct_customer_orders ............................... [RUN]
[2022-05-15 21:56:49,755] {subprocess.py:89} INFO - 21:56:49  1 of 1 OK created table model dev.fct_customer_orders .......................... [[32mSELECT 99[0m in 0.13s]
[2022-05-15 21:56:49,832] {subprocess.py:89} INFO - 21:56:49
[2022-05-15 21:56:49,834] {subprocess.py:89} INFO - 21:56:49  Finished running 1 table model in 0.40s.
[2022-05-15 21:56:49,837] {subprocess.py:89} INFO - 21:56:49
[2022-05-15 21:56:49,838] {subprocess.py:89} INFO - 21:56:49  [32mCompleted successfully[0m
[2022-05-15 21:56:49,840] {subprocess.py:89} INFO - 21:56:49
[2022-05-15 21:56:49,841] {subprocess.py:89} INFO - 21:56:49  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-05-15 21:56:50,876] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-15 21:56:50,902] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=model_level_dbt_dag, task_id=dbt_run.model.jaffle_shop.fct_customer_orders, execution_date=20220514T160047, start_date=20220515T215645, end_date=20220515T215650
[2022-05-15 21:56:50,944] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-15 21:56:50,992] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
